src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -36,10 +44,11 @@ const PageLayoutWrapper = ({
 
   const scriptSwitchId = pathOr('', ['scriptSwitchId'], pageData);
   const renderScriptSwitch = pathOr(true, ['renderScriptSwitch'], pageData);
-  const isErrorPage = [404, 500].includes(status);
-  const pageType = isErrorPage
-    ? 'WS-ERROR-PAGE'
-    : path<string>(['metadata', 'type'], pageData);
+
+  const isErrorPage = ![200].includes(status) || !status;
```

Comment:
Should we now revert this back to the original?

```suggestion
  const isErrorPage = [404, 500].includes(status);
```

Reply:
True. Let's leave your updated version - all pages that are working should have a status of 200, otherwise 💥